core/drivers/stm32_uart.c

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Minor: perhaps this should be introduced in a separate commit

Reply:
ok

Reply:
I will split commits for the platform from its uart driver was squashing the changes.